### DISCUSSION TITLE
Compare GitHub Discussions to Discourse

### DISCUSSION THREAD

**Discussion by cbf66** [arXiv/discussion#4] [General]
GitHub discussions

Pros:

- It's a product we already use
- Stable
- Simple UI

Cons:

- Not a good home for doc. So the doc and discussion are in separate places
- Limited ability to control participation
-- anyone on the internet can comment
-- only control for someone who misbehaves is to ban -- one at a time
-- miscreants can always create a new account and come back
- Tree control functionality is limited
-- Can't completely collapse a thread, which means a thread with a few replies will push subsequent topics off the screen

**Comment by bdc34** [comment]
I currently like the idea of a GH repo with discussions on GH. I could also see using issues to be resolved and then documents in the repo as the documentation. 

I'm hoping that while this isn't going to be perfect, it could be good enough. Especially since it is a low resource solution. Once discussions about arXiv-D hit up against limitations of either GH or Discord we can then spend resources to shift to a direction that handles the specific problems we encounter.
